20176.0.txt

```diff
@@ -0,0 +1,39 @@
+@0
+
+
+@1
+MOV UP, ACC
+SUB RIGHT
+MOV ACC, DOWN
+NEG
+MOV ACC, DOWN
+
+@2
+MOV UP, LEFT
+
+@3
+
+
+@4
+
+
+@5
+MOV UP, DOWN
+
+@6
+
+
+@7
+
+
+@8
+MOV UP, DOWN
+MOV UP, RIGHT
+
+@9
+MOV LEFT, DOWN
+
+@10
+
+
+#Used the accumulator again and used the NEG function for the firsty time.
```

Comment:
Comments should explain why each part of the program does what it does. I.e. what purpose does the `NEG` function serve here -- what does it achieve? Adding in-line comments to the code would help here.